src/screens/discover/components/DiscoverHome.js

```diff
@@ -20,12 +21,14 @@ import {
   cryptoAndWalletsCard,
 } from '@/components/cards/utils/constants';
 import { OpRewardsCard } from '@/components/cards/OpRewardsCard';
+import { LedgerCard } from '@/components/cards/LedgerCard';
 
 export default function DiscoverHome() {
   const { network } = useAccountSettings();
   const accountAsset = useAccountAsset(ETH_ADDRESS);
   const profilesEnabled = useExperimentalFlag(PROFILES);
   const optimismRewardsEnabled = useExperimentalFlag(OP_REWARDS);
+  const hardwareWalletsEnabled = useExperimentalFlag(HARDWARE_WALLETS);
```

Comment:
keep in mind useExperimentalFlag doesn't work in TestFlight, check my change here https://github.com/rainbow-me/rainbow/blob/f52a43af504c97810e155aa179d0a539fc40c0f0/src/config/experimentalHooks.ts#L8